help/sources/tutorials/ui/create/cloud-storage/merkury-enterprise-identity-resolution.md

```diff
@@ -0,0 +1,448 @@
+provides steps for creating a Merkury source connector using the
+Platform user interface.
+
+## Getting started
+
+This tutorial requires a working understanding of the following
+components of Adobe Experience Platform:
+
+-   [Experience Data Model (XDM)
+    System](https://experienceleague.adobe.com/docs/experience-platform/xdm/home.html?lang=en):
+    The standardized framework by which Experience Platform organizes
+    customer experience data.
+
+    -   [Basics of schema
+        composition](https://experienceleague.adobe.com/docs/experience-platform/xdm/schema/composition.html?lang=en):
+        Learn about the basic building blocks of XDM schemas, including
+        key principles and best practices in schema composition.
+
+    -   [Schema Editor
+        tutorial](https://experienceleague.adobe.com/docs/experience-platform/xdm/tutorials/create-schema-ui.html?lang=en):
+        Learn how to create custom schemas using the Schema Editor UI.
+
+-   [Real-Time Customer
+    Profile](https://experienceleague.adobe.com/docs/experience-platform/profile/home.html?lang=en):
+    Provides a unified, real-time consumer profile based on aggregated
+    data from multiple sources.
+
+If you already have a valid S3 connection, you may skip the remainder of
+this document and proceed to the tutorial on [configuring a
+dataflow](https://experienceleague.adobe.com/docs/experience-platform/sources/ui-tutorials/dataflow/cloud-storage.html?lang=en).
+
+### Gather required credentials
+
+In order to access your bucket on Platform, you need to provide valid
+values for the following credentials:
+
+  ---------------------------------------------------------------------------------
+  **Credential**   **Description**
+  ---------------- ----------------------------------------------------------------
+  AccessKey        The access key ID for your bucket.
+
+  SecretKey        The secret key ID for your bucket.
+
+  bucketName       The bucket contains your data and its corresponding descriptive
+                   metadata. Your S3 bucket name must be between three and 63
+                   characters long and must begin and end with either a letter or a
+                   number. The bucket name can only have lowercase letters,
+                   numbers, or hyphens (-), and cannot be formatted as an IP
+                   address.
+  ---------------------------------------------------------------------------------
+
+## Connect your Merkury Account
+
+In the Platform UI, select **Sources** from the left navigation bar to
+access the Sources workspace. The Catalog screen displays a variety of
+sources for which you can create an account with.
+
+You can select the Enrichment category from the catalog on the left-hand
+side of your screen. Alternatively, you can find the specific source you
+wish to work with using the search option.
+
+Under the **Enrichment** category, select **Merkury** and then
+select **Add data**.
```

Comment:
Looks like we will have a new Enrichment category, but the screenshot still uses Cloud Storage, can we get that updated?